### PR TITLE
refactor: Remove Balsamic container :wave: :wave:

### PR DESCRIPTION
**Pull request by ivadym**
### This PR:

The need for a Balsamic container has become redundant as the original plan to make it functional within the container isn't feasible. 

This container also slows the development process, as even small changes in constants require rebuilding it.

### Removed:
- Balsamic container

### Review and tests: 
- [x] Tests pass (Balsamic container dependant rules)
  - [x] Concatenation of fastq completed successfully
  - [x] Generated QC metrics file
  - [x] Rendered CNV report
  - [x] Analysis completed
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

File: .github/workflows/docker_build_push.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        container-name: [align_qc, annotate, coverage_qc, varcall_cnvkit, varcall_py27, varcall_py3, balsamic, delly, vcf2cytosure, cnvpytor, somalier, ascatNgs]
+        container-name: [align_qc, annotate, ascatNgs cnvpytor, coverage_qc, delly, somalier, varcall_cnvkit, varcall_py3, varcall_py27, vcf2cytosure]
```
Comment (mathiasbio) [inline .github/workflows/docker_build_push.yml:14]: Missing ","? 
```suggestion
        container-name: [align_qc, annotate, ascatNgs, cnvpytor, coverage_qc, delly, somalier, varcall_cnvkit, varcall_py3, varcall_py27, vcf2cytosure]
```

File: .github/workflows/docker_build_push_release.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        container-name: [align_qc, annotate, coverage_qc, varcall_cnvkit, varcall_py27, varcall_py3, balsamic, delly, vcf2cytosure, cnvpytor, somalier, ascatNgs]
+        container-name: [align_qc, annotate, ascatNgs cnvpytor, coverage_qc, delly, somalier, varcall_cnvkit, varcall_py3, varcall_py27, vcf2cytosure]
```
Comment (mathiasbio) [inline .github/workflows/docker_build_push_release.yml:13]: Missing ","?
```suggestion
        container-name: [align_qc, annotate, ascatNgs, cnvpytor, coverage_qc, delly, somalier, varcall_cnvkit, varcall_py3, varcall_py27, vcf2cytosure]
```

**Review by mathiasbio — APPROVED** [review]
Nice! 
⭐ 🧙💬 Avada Kedavra! 🟢  ----> balsamic.sif  💀
Good riddance!

File: .github/workflows/docker_build_push.yml (hunk 1 above)
Comment (ivadym) [reply .github/workflows/docker_build_push.yml:14]: Oh nice catch! You have an eye of an eagle 🦅

File: .github/workflows/docker_build_push.yml [hunk 3]
```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        container-name: [align_qc, annotate, coverage_qc, varcall_cnvkit, varcall_py27, varcall_py3, balsamic, delly, vcf2cytosure, cnvpytor, somalier, ascatNgs]
+        container-name: [align_qc, annotate, ascatNgs, cnvpytor, coverage_qc, delly, somalier, varcall_cnvkit, varcall_py3, varcall_py27, vcf2cytosure]
```
Comment (fevac) [inline .github/workflows/docker_build_push.yml:14]: is there a shortcut to sort lists in pycharm?

File: BALSAMIC/constants/cache.py [hunk 4]
```diff
@@ -42,7 +42,6 @@ class DockerContainers(StrEnum):
     ALIGN_QC: str = "align_qc"
     ANNOTATE: str = "annotate"
     ASCAT: str = "ascatNgs"
-    BALSAMIC: str = "balsamic"
```
Comment (fevac) [inline BALSAMIC/constants/cache.py:45]: 👋 👋

**Review by fevac — APPROVED** [review]
Hasta la vista, Baby! Great job! 🌟

File: BALSAMIC/constants/cache.py (hunk 4 above)
Comment (ivadym) [reply BALSAMIC/constants/cache.py:45]: 🌈

**Pull request merged**